### ISSUE TITLE
question about updating existing dxf

### ISSUE THREAD

**Issue by he-yang** [dxfjs/writer#12]
hi, i am working on an application which can automatically add text to existing dxf drawings.

I notice your application "dxf" can create new dxf file, would it be possible if you read existing dxf files first and then I put some new TEXTs to the dxf file, later export as a new dxf ?

**Comment by tarikjabiri** [comment]
Hi

Yeah currently the lib generate only new dxf content 

I am planing to add a dxf parser to the lib in the future. For that purpose a rewrite of the lib is needed.

**Comment by he-yang** [comment]
wow. I just noted you are moving forward to 2.0.0. Great.

**Comment by tarikjabiri** [comment]
Yeah it's already v2.1.0, I have a new repo parser but it's not yet completed.

**Comment by DeepankarGupta13** [comment]
Hi, I am using text but I am not able to rotate the text.
can you pls help me with this?

**Comment by DeepankarGupta13** [comment]
the above issue is fixed by @tarikjabiri .

**Comment by sonikdesign** [comment]
Hi, what is the lastest update on this feature?

**Comment by tarikjabiri** [comment]
Actually no update here, but there is a seprate repo for parsing dxf, you can use that to parse the dxf and use the writer to write after

**Comment by 497983606** [comment]
> 实际上这里没有更新，但是有一个单独的 repo 用于解析 dxf，你可以使用它来解析 dxf 并使用 writer 进行写入

there doesn't seem to be any indication of how you can quickly display the parsed content back on the page, please reply

**Comment by tarikjabiri** [comment]
Hi, 

You can't display dxf using my libs, I mean the writer and parser, if you want to display the content of the dxf you can use [https://github.com/gdsestimating/three-dxf](https://github.com/gdsestimating/three-dxf). take a look here [https://github.com/vagran/dxf-viewer](https://github.com/vagran/dxf-viewer).

> [!NOTE]
> This lib can only write dxf content, hence creating "valid" dxf file.
